README.md

```diff
@@ -92,6 +92,7 @@ More libraries are coming soon (the overall list of items we currently plan to m
 |**System.ObjectModel**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.ObjectModel.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides types and interfaces that enable the creation of observable types that provide notifications to clients when changes are made.</sub>|
 |**System.Reflection**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides types that retrieve information about assemblies, modules, members, parameters, and other entities in managed code by examining their metadata.</sub>|
 |**System.Reflection.DispatchProxy**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.DispatchProxy.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides a mechanism for dynamically creating proxy types that implement a specified interface and derive from a specified DispatchProxy type.</sub>|
+|**System.Reflection.Emit.ILGeneration**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Reflection.Emit.ILGeneration.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides clases that allow a compiler or other tool to emit Microsoft intermediate language (MSIL).</sub>|
```

Comment:
Thanks, will fix. I copied this text directly from https://www.nuget.org/packages/System.Reflection.Emit.ILGeneration/, where the typo apparently also exists :)